setup.py

```diff
@@ -25,7 +25,12 @@ def package_files(directory):
     url="https://github.com/equinor/semeio",
     description="Jobs and workflow jobs for Ert.",
     packages=find_packages(include=["semeio*"], exclude=["semeio.jobs.scripts"]),
-    entry_points={"ert": ["semeio_jobs = semeio.hook_implementations.jobs"]},
+    entry_points={
+        "ert": [
+            "semeio_jobs = semeio.hook_implementations.jobs",
+            "MisfitPreprocessor = semeio.jobs.scripts.misfit_preprocessor",
```

Comment:
Why do we need to have this line here? Isn't it redundant with `scripts = glob("semeio/jobs/scripts/[!__]*.py")`?

Reply:
We probably could, but I am not convinced we should do it implicitly. The main drawbacks in my view is having it explicitly allows anyone to quickly tell how many entry points we have, and where they are. The second is that by `glob`ing, we have to keep all the jobs in the same place. I would prefer we dont any more, and put everything related to one workflow in a separate module. That makes it much easier to navigate.

Reply:
I understand, do you think we could let explicit the other jobs declared in the glob, to be consistent overall? 
Maybe in a different commit, but same PR?

Reply:
I am not quite sure I understand what you mean? Do you mean the forward model jobs? The other workflows have to add their own entry point.

Reply:
I see. As we are introducing a new way to define entry points, I thought it would follow with the modification of the other entry points (so all have the same standard).

Reply:
I think we should do that to forward model jobs as well, but there is no system for that in `ert`, yet. For now we will do it for all the workflow jobs, but they will add their own entry point as they are updated from the old to the new spec.